src/hooks/api/announceAuction.ts

```diff
@@ -65,7 +66,8 @@ export const useAnnounceAuction = (
         queryKey: [QUERY_AUCTIONS_QUERY_KEY, config],
       });
     },
-    onError: (error) => {
+    onError: (error: Error, params) => {
+      trackError(error, 'announceAuction', mixPanel, params);
```

Comment:
Any issue with making `onError` a util you can just import everywhere instead of redefining it in each file?

Reply:
Since we are calling these three functions (trackError, toast.error, and console.error) on each onError, we can make a helper function called `logError` which takes error` and `param` as arguments and calls these 3 lines. good point!